Payara-Micro/spring-boot-jpa-jsf-example/README.md

```diff
@@ -0,0 +1,11 @@
+Compile with:
+
+    mvn compile
```

Comment:
It'd be better to address install phase as: mvn install, instead of mvn compile since it's the phase that gets war file built.